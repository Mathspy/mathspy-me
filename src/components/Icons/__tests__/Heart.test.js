import React from "react";

import { shallow } from "enzyme";

import Heart from "../Heart";

describe("Heart", () => {
  it("should render Heart's icon!", () => {
    const wrapper = shallow(<Heart />);

    expect(wrapper).toMatchSnapshot();
  });

  it("should pass all properties to the root of the svg", () => {
    const wrapper = shallow(<Heart test="test" random={123} />);

    expect(wrapper.props()).toMatchObject({ test: "test", random: 123 });
  });
});
