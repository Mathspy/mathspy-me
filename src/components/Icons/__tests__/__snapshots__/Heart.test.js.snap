// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heart should render Heart's icon! 1`] = `
<svg
  version="1"
  viewBox="0 0 645 585"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M297 551a668 668 0 0 0-76-67c-84-63-95-72-129-104C29 323 2 265 3 186c0-38 2-53 13-76 18-38 45-67 79-84 25-13 37-18 77-18 43-1 52 4 77 18 30 17 62 53 68 78l4 16 10-22c56-122 233-120 295 3 20 39 22 123 5 170-23 61-66 108-164 179a937 937 0 0 0-143 128c-6 12-1 2-27-27z"
    fill="#FFF"
  />
</svg>
`;
